tests.php

```diff
@@ -0,0 +1,114 @@
+<?php
+	/* Tests script for phpcs-security-audit */
+
+	//eval($_GET['a']);
+	echo "aaaa";
+	echo "bbbb" . $_POST['b'];
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> error - Easy XSS detected because of direct user input with $_POST on echo